rest-api Create rest api example

Davvidos opened the pull request:
Remove some files from idea dir. Create new 404 json handler. Create
article controller. Create default article model. Create migration for
articles. Create default seeder with articles. Create routes for api
endpoints.

Davvidos committed:
rest-api Create rest api example

Remove some files from idea dir. Create new 404 json handler. Create
article controller. Create default article model. Create migration for
articles. Create default seeder with articles. Create routes for api
endpoints.

jgrygierek commented on composer.json:
```diff
@@ -6,6 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.3|^8.0",
```
What is purpose of multiple php versions here? You use just one of them. If your app is used by more people with different php versions - ok, but otherwise I don't see here any sense. But if you want to have multiple php versions - do not forget about testing your code for each of them.

Davvidos replied:
@jgrygierek Please check it now.

Davvidos committed:
rest-api Create rest api example

Update readme file. Update php version. Add notes with instruction to
do tasks.
